api/core/domain/integration_domain_test.py

```diff
@@ -1,8 +1,9 @@
-from core.domain.integration_domain import (
+from core.domain.integration_domain.integration_domain import (
```

Comment:
this should be moved to the `integration_domain` dir ?

Also the directory should likely be named `integrations` ?

Reply:
fixed